components/scalr/scalr.app.mjs

```diff
@@ -0,0 +1,73 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "scalr",
+  propDefinitions: {
+    accountId: {
+      type: "string",
+      label: "Account ID",
+      description: "The ID of the account you wish to use.",
+      async options() {
+        const { data: accounts } = await this.getAccounts();
+        return accounts.map((account) => ({
+          label: account.attributes.name,
+          value: account.id,
+        }));
+      },
```

Comment:
We should use pagination in async options whenever possible. The [documentation](https://docs.scalr.io/reference/get_accounts) shows that the list accounts endpoint accepts `page[number]` as a parameter.